services/src/main/scala/cromwell/services/ServiceRegistryActor.scala

```diff
@@ -107,6 +110,15 @@ class ServiceRegistryActor(globalConfig: Config) extends Actor with ActorLogging
       sender() ! ServiceRegistryFailure("Message is not a ServiceRegistryMessage: " + fool)
   }
 
+  private def debugLogLoadMessages(msg: ServiceRegistryMessage, sender: ActorRef): Unit = {
+    msg match {
```

Comment:
optional thought: Maybe we'd prefer to be logging the `LoadMetric`s themselves (or some `toString` method on them) rather than making custom messages based on their `loadLevel` here. Slightly more robust in the case that we add new `LoadMetric`s

Reply:
I guess I wrote the code this way when I wanted different log levels for the two, but they're both `debug` now. Implemented.

```
> amm
Loading...
Welcome to the Ammonite Repl 2.5.11 (Scala 3.2.2 Java 11.0.20.1)
@ trait LoadLevel 
defined trait LoadLevel

@ case object NormalLoad extends LoadLevel 
defined object NormalLoad

@ val x = NormalLoad 
x: ammonite.$sess.cmd1.NormalLoad.type = NormalLoad

@ x.toString 
res3: String = "NormalLoad"
```